android/src/main/java/com/rnappauth/RNAppAuthModule.java

```diff
@@ -884,7 +886,7 @@ private AppAuthConfiguration createAppAuthConfiguration(
     /*
      *  Create appropriate connection builder based on provided settings
      */
-    private ConnectionBuilder createConnectionBuilder(boolean allowInsecureConnections, Map<String, String> headers) {
+    private ConnectionBuilder createConnectionBuilder(boolean allowInsecureConnections, Map<String, String> headers, Double connectionTimeoutMillis) {
```

Comment:
The android build fails for me. Perhaps you forgot to copy over the code that adds the `connectionTimeoutMillis ` to createConnectionBuilder on the Android side?
```
23 actionable tasks: 2 executed, 21 up-to-date
/Users/kadi/Code/react-native-app-auth/Example/node_modules/react-native-app-auth/android/src/main/java/com/rnappauth/RNAppAuthModule.java:105: error: method createConnectionBuilder in class RNAppAuthModule cannot be applied to given types;
        final ConnectionBuilder builder = createConnectionBuilder(dangerouslyAllowInsecureHttpRequests, this.authorizationRequestHeaders);
                                          ^
  required: boolean,Map<String,String>,Double
  found: boolean,Map<String,String>
  reason: actual and formal argument lists differ in length
/Users/kadi/Code/react-native-app-auth/Example/node_modules/react-native-app-auth/android/src/main/java/com/rnappauth/RNAppAuthModule.java:164: error: method createConnectionBuilder in class RNAppAuthModule cannot be applied to given types;
        final ConnectionBuilder builder = createConnectionBuilder(dangerouslyAllowInsecureHttpRequests, this.registrationRequestHeaders);
                                          ^
  required: boolean,Map<String,String>,Double
  found: boolean,Map<String,String>
  reason: actual and formal argument lists differ in length
/Users/kadi/Code/react-native-app-auth/Example/node_modules/react-native-app-auth/android/src/main/java/com/rnappauth/RNAppAuthModule.java:237: error: method createConnectionBuilder in class RNAppAuthModule cannot be applied to given types;
        final ConnectionBuilder builder = createConnectionBuilder(dangerouslyAllowInsecureHttpRequests, this.authorizationRequestHeaders);
                                          ^
  required: boolean,Map<String,String>,Double
  found: boolean,Map<String,String>
  reason: actual and formal argument lists differ in length
/Users/kadi/Code/react-native-app-auth/Example/node_modules/react-native-app-auth/android/src/main/java/com/rnappauth/RNAppAuthModule.java:329: error: method createConnectionBuilder in class RNAppAuthModule cannot be applied to given types;
        final ConnectionBuilder builder = createConnectionBuilder(dangerouslyAllowInsecureHttpRequests, this.tokenRequestHeaders);
                                          ^
  required: boolean,Map<String,String>,Double
  found: boolean,Map<String,String>
  reason: actual and formal argument lists differ in length
/Users/kadi/Code/react-native-app-auth/Example/node_modules/react-native-app-auth/android/src/main/java/com/rnappauth/RNAppAuthModule.java:412: error: method createConnectionBuilder in class RNAppAuthModule cannot be applied to given types;
        final ConnectionBuilder builder = createConnectionBuilder(dangerouslyAllowInsecureHttpRequests, null);
                                          ^
  required: boolean,Map<String,String>,Double
  found: boolean,<null>
  reason: actual and formal argument lists differ in length
/Users/kadi/Code/react-native-app-auth/Example/node_modules/react-native-app-auth/android/src/main/java/com/rnappauth/RNAppAuthModule.java:507: error: method createConnectionBuilder in class RNAppAuthModule cannot be applied to given types;
                    createConnectionBuilder(this.dangerouslyAllowInsecureHttpRequests, this.tokenRequestHeaders),
                    ^
  required: boolean,Map<String,String>,Double
  found: boolean,Map<String,String>
  reason: actual and formal argument lists differ in length
6 errors
```

Reply:
Ah, that makes sense!